submissions/lalalilulu/friends-app/script.js

```diff
@@ -0,0 +1,139 @@
+document.addEventListener("DOMContentLoaded", () => {
+    fetchContent().then((response) => startApp(response.results)).catch(error => console.log(error.message));
+})
+
+const startApp = (response) => {
+    renderContent(response);
+    const friendsArray = response;
+
+    document.querySelector("#filter-form").addEventListener("input", ({target}) => {
+        let sortedArray = [];
+        switch (target.id) {
+            case "search":
+                sortedArray = searchByValue(friendsArray, target.value);
+                break;
+            case "name-ascending":
+                sortedArray = sortByName(friendsArray, true);
+                break;
+            case "name-descending":
+                sortedArray = sortByName(friendsArray, false);
+                break;
+            case "age-ascending":
+                sortedArray = sortByAge(friendsArray, true);
+                break;
+            case "age-descending":
+                sortedArray = sortByAge(friendsArray, false);
+                break;
+            case "female":
+                resetCheckedFilters();
+                sortedArray = sortByGender(friendsArray, "female");
+                break;
+            case "male":
+                resetCheckedFilters();
+                sortedArray = sortByGender(friendsArray, "male");
+                break;
+            case "all":
+                resetCheckedFilters();
+                sortedArray = sortByGender(friendsArray, null);
+                break;
```

Comment:
Do you really need to call this function here?
in this case it just returns the friendsArray unchanged.